### DISCUSSION TITLE
Coding help

### DISCUSSION THREAD

**Discussion by TheGreatJG** [community/community#141492] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Yo. I’m not really good at coding and was wondering if someone could make me a deployable proxy using ultraviolet. I wanna have it look like chrome with tabs, reload, and back button as well as using local history for saving history and website data stuff. Also, in its browser settings I want it to have a tab cloak feature where it asks for url and then grabs the fav icon and uses that. Also I want it to have a setable panic button . Last,y, if not able to make deployable to for instance Vercel, can you make it so when using replit to run it creates a json to use to store stuff? Sorry for so much to ask. Thanks y’all!

**Comment by LiteBrite82** [comment]
Hi @TheGreatJG , we’re glad you’re here 😀

Thanks for sharing your project idea with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
